### PR TITLE
Update the_form_view.rst

### PR DESCRIPTION
**Pull request by gutigrewal**
I am targeting this branch, because the documentation for the The Form View is not quite correct.

### CODE REVIEW

File: docs/getting_started/the_form_view.rst [hunk 1]
```diff
@@ -157,14 +159,16 @@ category field to a Meta data group. To do this, use the ``with()`` method:
 .. code-block:: php
 
     // src/AppBundle/Admin/BlogPostAdmin.php
-
+	use Symfony\Component\Form\Extension\Core\Type\TextareaType;
```
Comment (greg0ire) [inline docs/getting_started/the_form_view.rst]: Indention issue

**Comment by greg0ire** [comment]
I removed the changelog because it is not needed for documentation PRs

**Comment by kunicmarko20** [comment]
Can you do this for the rest of the forms in this document?

**Comment by jordisala1991** [comment]
Duplicated of this other #4976

**Pull request closed without merging**